oxidation/client/electron/src/setup.ts

```diff
@@ -165,7 +165,7 @@ export class ElectronCapacitorApp {
     // Setup the main manu bar at the top of our window.
     Menu.setApplicationMenu(Menu.buildFromTemplate(this.AppMenuBarMenuTemplate));
 
-    /* If the splashscreen is enabled, show it first while the main window loads then dwitch it out for the main window,
+    /* If the splashscreen is enabled, show it first while the main window loads then ditch it out for the main window,
```

Comment:
What about this comment:
```
If the splash screen is enabled, show it first while the main window loads, then dismiss it to display the main window. Alternatively, you can just load the main window from the start without showing the splash screen.
```

Reply:
The goal was to correct typo, not to rewrite comment ;)

Reply:
But at least that would prevent the typo, I'll change it

Reply:
done